### DISCUSSION TITLE
remove this virus

### DISCUSSION THREAD

**Discussion by CodeBhaukaal** [capcom6/android-sms-gateway#98] [General]
Android.Malware.General (score:7)

![screenshot-www virustotal com-2024 07](https://github.com/user-attachments/assets/c2bc0c58-2c9b-4e84-8e1b-f5610b64c66a)
plz remove this

**Comment by capcom6** [comment]
A single detection out of 68 antiviruses suggests a false positive. The app does not contain any malware. Since it is open-source software, you are welcome to review the code yourself. If you have questions about specific parts, feel free to ask.

I suspect the detection might result from the use of unconventional features in the app, such as starting a local web server and sending SMS messages in the background. However, these are documented features necessary for the app's proper functionality.

**Reply by k9protection** [reply]
Also flagged on my system. Works a charm and isn't doing anything nefarious.
